src/stats/steam.go

```diff
@@ -35,7 +34,7 @@ type DataContainer struct {
 
 func fetchSteam(id int) (int, error) {
 	url := BASEURL + strconv.Itoa(id)
-	log.Println(fmt.Sprintf("[STEAM] Fetching current player count on %s", url))
```

Comment:
Fetch logs left out for now as gcloud logging changes not applied here yet
Best to do it when code structure finalised